packages/core-sdk/src/types/resources/license.ts

```diff
@@ -202,3 +196,12 @@ export type SetLicensingConfigResponse = {
   encodedTxData?: EncodedTxData;
   success?: boolean;
 };
+
+export type GetLicensingConfigRequest = {
+  /** The address of the IP for which the configuration is being set. */
+  ipId: Address;
+  /** The ID of the license terms within the license template. */
+  licenseTermsId: string | number | bigint;
+  /** The address of the license template used. */
+  licenseTemplate: Address;
```

Comment:
i think we should make this optional and default to `piLicenseTemplateAddress` of the current chain if not provided.